src/SFA.DAS.EAS.Web/Views/Shared/EditApprenticeship.cshtml

```diff
@@ -77,64 +78,76 @@ else
     </div>
 }
 
-<div class="form-group @(!string.IsNullOrEmpty(Model.Apprenticeship.TrainingCodeError) ? "error" : "")">
-    <hr />
-    <label class="form-label-bold" for="TrainingCode">Apprenticeship training course</label>
-    <span class="form-hint">Start typing in the name of the course or choose an option from the list</span>
-    @if (!string.IsNullOrEmpty(Model.Apprenticeship.TrainingCodeError))
-    {
-        <span class="error-message" id="error-message-TrainingCode">Choose a training course for this apprentice</span>
-    }
-    <select name="TrainingCode" id="TrainingCode" class="form-control form-control-3-4" aria-label="Apprenticeship training course">
-        <option value="">Please select</option>
-        @foreach (var apprenticeshipProduct in Model.ApprenticeshipProgrammes)
+    <div class="form-group @(!string.IsNullOrEmpty(Model.Apprenticeship.TrainingCodeError) ? "error" : "")">
+        <hr />
+        <label class="form-label-bold" for="TrainingCode">Apprenticeship training course</label>
+        @if (Model.Apprenticeship.IsTransferFundedAndNoSuccessfulIrlSubmission)
         {
-            <option value="@apprenticeshipProduct.Id" @if (apprenticeshipProduct.Id == Model.Apprenticeship.TrainingCode) { @Html.Raw("selected")    }>
+            <span id="TrainingCode">@Model.Apprenticeship.TrainingName</span>
+            @Html.Hidden("TrainingCode", Model.Apprenticeship.TrainingCode)
+        }
+        else
+        {
+            <span class="form-hint">Start typing in the name of the course or choose an option from the list</span>
+            if (!string.IsNullOrEmpty(Model.Apprenticeship.TrainingCodeError))
+            {
+                <span class="error-message" id="error-message-TrainingCode">Choose a training course for this apprentice</span>
+            }
+            <select name="TrainingCode" id="TrainingCode" class="form-control form-control-3-4" aria-label="Apprenticeship training course">
+            <option value="">Please select</option>
+            @foreach (var apprenticeshipProduct in Model.ApprenticeshipProgrammes)
+            {
+                <option value="@apprenticeshipProduct.Id" @if (apprenticeshipProduct.Id == Model.Apprenticeship.TrainingCode) { @Html.Raw("selected") }>
                 @apprenticeshipProduct.Title
-            </option>
+                </option>
+            }
+            </select>
         }
-    </select>
-
-</div>
+    </div>
 
 <div class="form-error-group form-group @(!string.IsNullOrEmpty(Model.Apprenticeship.StartDateError) || !string.IsNullOrEmpty(Model.Apprenticeship.StartDateOverlapError) ? "error" : "")">
     <hr />
 
     <span class="form-label-bold">Planned training start date</span>
-    <span class="form-hint">For example, 09 2017</span>
 
-    <div id="StartDate" class="form-date">
+    @if (Model.Apprenticeship.IsTransferFundedAndNoSuccessfulIrlSubmission)
```

Comment:
I know this is unlikely, but do we ever have malicious users. i.e. are we checking the hidden field values have not been changed by the user? The user in theory could change values and submit the form (very unlikely), and hence avoid business rules unless we write them in the back-end as well

Reply:
I think we should assume we have malicious users!

It's probably beyond the scope of this story to guard against it throughout the sites and I'm not sure guarding about it here, but not everywhere is the thing to do, but it might be worthwhile to add a story to audit and fix the two websites.